tests/selenium/spec/header-spec.js

```diff
@@ -34,10 +35,16 @@ describe('header sanity check', () => {
         .to.contain("/signup/");
     }));
 
-    it('pricing link click takes user to pricing page', util.itHelper(async () => {
+    it('pricing link click takes user to pricing okta.com page from a new tab', util.itHelper(async () => {
+      const pricingURL = 'https://www.okta.com/pricing/#customer-identity-products';
+
       await mainPage.getPricingLinkElement().click();
-      expect(await mainPage.getCurrentURL())
-        .to.contain("/pricing/");
+      await browser.sleep(10000);
```

Comment:
How did we choose 10 seconds?  Wouldn't the wait method be a better solution here. Network or performance issues could make the wait more than 10 seconds. Also, we are adding another 10 seconds to be build process (>45 mins) when it could be less time depending on network, cpu, etc.  For more info on the wait method see:

https://riptutorial.com/protractor/example/26620/browser-sleep---vs-browser-wait--
https://stackoverflow.com/questions/45584840/what-is-the-difference-between-browser-sleep-and-browser-wait-methods

Reply:
I've made some edits, `sleep` method doesn't require here